docker/scripts/p4app.py

```diff
@@ -0,0 +1,89 @@
+from mininet.net import Mininet
+from mininet.topo import Topo, SingleSwitchTopo
+
+from p4_mininet import P4Host, P4Switch
```

Comment:
you don't seem to be using P4Switch anywhere?

Reply:
`P4Switch` is intended to be used with `simple_switch` (i.e. the non-p4runtime target). However, it is not currently exposed. I will replace this class with `P4RuntimeSwitch` and make it compatible with both targets: `simple_switch` and `simple_switch_grpc`.

Reply:
`P4Switch` is from the old p4app and is for the `simple_switch` target, while `P4RuntimeSwitch` is from p4lang/tutorials and is for the `simple_switch_grpc` target. `P4Switch` is not currently exposed, but I think we should keep it if we want to support the `simple_switch` target.